lib/activerecord-import/import.rb

```diff
@@ -633,6 +648,34 @@ def import_associations(models, options)
       end
     end
 
+    def import_habtm(habtm_models)
+      habtm_inserts = {}
+      habtm_models.group_by(&:class).each do |klass, models|
+        habtms = habtm_reflections(klass)
+
+        next if habtms.blank?
+
+        habtms.each do |reflection|
+          models.each do |model|
+            association_ids = model.send(reflection.name).collect(&:id)
+            next if association_ids.blank?
+            habtm_inserts[reflection.association_foreign_key] ||= [reflection, []]
+            habtm_inserts[reflection.association_foreign_key].last.push(*association_ids.zip([model.id] * association_ids.count))
+          end
+        end
+      end
+
+      habtm_inserts.each_value do |reflection, values|
+        join_table = reflection.options[:join_table] || reflection.join_table
+        insert_sql = prepare_insert_sql([reflection.association_foreign_key, reflection.foreign_key], table_name: join_table)
+        perform_insert insert_sql, values_sql_for_attributes(values)
```

Comment:
Just reviewing this, I'm not able to wrap my head around what `habtm_inserts` contains. Could you explain that and how many insert queries this will execute?